modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,395 @@
+|Indicates whether the OSUS graph is added to the image set and subsequently published to the mirror.
+|Boolean. The default value is `false`.
+
+|===
+
+[NOTE]
+====
+Using the `minVersion` and `maxVersion` properties to filter for a specific Operator version range can result in a multiple channel heads error. The error message states that there are `multiple channel heads`. This is because when the filter is applied, the update graph of the Operator is truncated.
+Operator Lifecycle Manager requires that every Operator channel contains versions that form an update graph with exactly one end point, that is, the latest version of the Operator. When the filter range is applied, that graph can turn into two or more separate graphs or a graph that has more than one end point.
+To avoid this error, do not filter out the latest version of an Operator. If you still run into the error, depending on the Operator, either the `maxVersion` property must be increased or the `minVersion` property must be decreased. Because every Operator graph can be different, you might need to adjust these values until the error resolves.
+====
+
+[id="delete-imagset-config-parameters"]
+== Delete Image Set Configuration parameters
+
+The oc-mirror plugin requires an delete image set configuration file that defines what images to delete from the mirror registry. The following table lists the available parameters for the `DeleteImageSetConfiguration` resource.
+
+.`DeleteImageSetConfiguration` parameters
+[cols="2,2a,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `DeleteImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v2alpha1`.
+
+|`delete`
+|The configuration of the image set to delete.
+|Object
+
+|`delete.additionalImages`
+|The additional images configuration of the delete image set.
+|Array of objects. For example:
+[source,yaml]
+----
+additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+----
+
+|`delete.additionalImages.name`
+|The tag or digest of the image to delete.
+|String. For example: `registry.redhat.io/ubi8/ubi:latest`
+
+|`delete.operators`
+|The Operators configuration of the delete image set.
+|Array of objects. For example:
+[source,yaml]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:vBranch Build
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '2.4.0'
+----
+
+|`delete.operators.catalog`
+|The Operator catalog to include in the delete image set.
+|String. For example: `registry.redhat.io/redhat/redhat-operator-index:v4.15`.
+
+|`delete.operators.full`
+|When true, deletes the full catalog, Operator package, or Operator channel.
+|Boolean. The default value is `false`.
+
+|`delete.operators.packages`
+|The Operator packages configuration.
+|Array of objects. For example:
+[source,yaml]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:vBranch Build
```

Comment:
why BranchBuild

Reply:
It will be replaced with {product-version}